specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
I didn't understand why we should have for example 
`specs/data/transactions/may/transactions.csv`
the program now doesn't search for csv files in any folder expect in the files folder and the root folder